src/procedures/ReclaimFunds.ts

```diff
@@ -18,9 +21,18 @@ export const createReclaimFundsResolver = (
   );
 };
 
+/**
+ * Procedure that allows the issuer reclaim remaining unclaimed dividend amounts, for expired dividends
```

Comment:
Procedure that allows the issuer to reclaim dividends after they expire without being claimed by shareholders